VOXELPLA PLUS BLACK.xml.fdm_material

```diff
@@ -0,0 +1,252 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+  <metadata>
+    <name>
+      <brand>VOXELPLA</brand>
+      <material>PLA</material>
+      <color>BLACK</color>
+      <label>VOXELPLA PLUS BLACK</label>
+    </name>
+    <version>8</version>
+    <GUID>da7d7354-792f-4e80-ac3b-59df1de531fa</GUID>
+    <color_code>#000000</color_code>
+    <description>The most reliable PLA PLUS on the planet. Best printed with 200C on the extruder and 45-60C on the bed, minimal retraction is needed. Each spool is $16.99 on WWW.VOXELPLA.COM</description>

```

Comment:
This description is not helpful for users. 
The bold claim that this is The Most Reliable PLA PLUS on the planet cannot be fact-checked.
Including the price and website is not a good idea since it's subject to change and can only be updated when shipped with a Cura release. 
I suggest removing these mentions and making them a more nuanced message.